src/controllers/organisations.ts

```diff
@@ -37,16 +44,35 @@ interface UpdateOrganisationRequestSchema {
     website_url?: string;
     phone?: string;
     email?: string;
+    address?: string;
     locations?: string[];
     story?: string;
+    facebook_link?: string;
+    twitter_link?: string;
+    instagram_link?: string;
+    banner_photo?: string;
+    profile_photo?: string;
+    additional_photos?: string[];
 }
 
 /**
  * @desc    Get all organisations
  * @route   GET /api/organisations
+ * @desc    Get all organisations for a single listing
+ * @route   GET /api/listings/:listing_id/organisations
  * @access  Public
  */
 export const getOrganisations = asyncHandler(async (req, res) => {
+    if (req.params.listing_id) {
+        const rows = await db.manyOrNone<Promise<OrganisationSchema[]>>(
```

Comment:
This db query will not return <Promise<OrganisationSchema[]>> as it joins  `listing_organisations` and `listings`. We can leave the type definition for this particular call for now, since we are refactoring the type definitions.

It will probably be a combination of listings and organisations schema

```
interface ListingOrganisationSchema: ListingSchema & OrganisationSchema
```

`&` - see https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html#intersection-types